### DISCUSSION TITLE
Need help, to get my PM session Id

### DISCUSSION THREAD

**Discussion by ArneHH79** [apideck-libraries/postman-to-k6#100] [Q&A] [answered]
Hi everyone, I have a Postman Collection that I exported. I know pm.response.headers is not supported when converting. Does anyone have an idea how I can still get my session ID? 

Here just my simple PM tests: 

if(pm.response.code !== 200){
	console.error("statuscode:", pm.response.code, ", response body: " , pm.response.text());
}
pm.test("Status code is 200", function () {
	pm.response.to.have.status(200);
});

pm.globals.set("Session ID", pm.response.headers.get("SESSION"));

**Comment by ArneHH79** [comment]
Any Solution?

**Reply by thim81** [reply, marked as answer]
hi @ArneHH79 

Previously postman-to-k6 did not support `pm.response.headers`. 
Since [version 1.9.0](https://github.com/apideck-libraries/postman-to-k6/releases/tag/v1.9.0), we have introduced the methods: pm.response.headers.get("xyz") and pm.response.headers.all().

This should help you overcome your problem.
